### PR TITLE
[jk] Add button for installing  packages from requirements.txt file

### PR DESCRIPTION
**Pull request by johnson-mage**
# Summary
- When user clicks on `requirements.txt` file in root project folder, they can pip install the packages saved in that file by clicking on a button (instead of having to enter a command through the Sidekick terminal). This automatically opens up the terminal to see the output from the `pip install -r requirements.txt` command.

# Tests
pip install packages

cc:
@tommydangerous

### CODE REVIEW

File: mage_ai/frontend/components/FileEditor/index.tsx [hunk 1]
```diff
@@ -215,6 +230,52 @@ function FileEditor({
...
+
+  const installPackagesButtonEl = (
+    <Spacing m={2}>
+      <KeyboardShortcutButton
+        inline
+        loading={loading}
+        onClick={() => {
+          openSidekickView(ViewKeyEnum.TERMINAL);
+          sendMessage(JSON.stringify({
+            code: `!pip install -r ~/../home/src/${projectName}/requirements.txt`,
```
Comment (johnson-mage) [inline mage_ai/frontend/components/FileEditor/index.tsx:268]: @wangxiaoyou1993 When user clicks the "Install packages" button, we run this command. Is this path reliable for getting to the project's `requirements.txt` file?

Comment (wangxiaoyou1993) [reply mage_ai/frontend/components/FileEditor/index.tsx:268]: it's not always this path. at backend, we get the path via
```
from mage_ai.data_preparation.repo_manager import get_repo_path
`get_repo_path()/requirements.txt`
```

File: mage_ai/server/websocket.py [hunk 2]
```diff
@@ -183,6 +183,10 @@ def on_message(self, raw_message):
             # Need to use Python magic command for changing directories
             code = re.sub(r'^!%cd|^!cd', '%cd', code)
 
+            install_packages = message.get('install_packages')
+            if install_packages:
+                code = f'!pip install -r {get_repo_path()}/requirements.txt'
```
Comment (johnson-mage) [inline mage_ai/server/websocket.py:188]: @wangxiaoyou1993 Does using a flag in the request message work? Or is there a better way?

Comment (wangxiaoyou1993) [reply mage_ai/server/websocket.py:188]: It doesn't work. `get_repo_path()` can't be used in bash command. You can get repo_path from backend in one API. For example, include the repo_path in `/api/status` https://github.com/mage-ai/mage-ai/blob/master/mage_ai/server/server.py#L417, and then use that in the command.

**Comment by johnson-mage** [comment]
@wangxiaoyou1993 Okay, I think it should be good now. Can you re-review please?

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**